src/NUnitEngine/nunit.engine/Internal/ResultHelper.cs

```diff
@@ -83,10 +89,10 @@ public static TestEngineResult MakePackageResult(this TestEngineResult result, s
         /// <remarks>Used by AbstractTestRunner MakePackageResult method.</remarks>
         public static TestEngineResult Merge(IList<TestEngineResult> results)
         {
-            TestEngineResult mergedResult = new TestEngineResult();
+            var mergedResult = new TestEngineResult();
 
-            foreach (TestEngineResult result in results)
-                foreach (XmlNode node in result.XmlNodes)
+            foreach (var result in results)
+                foreach (var node in result.XmlNodes)
```

Comment:
I am fine with the previous 2 vars, but think that this one decreases readability. It probably won't happen in this case, but if ever in the future someone needs to use `node`, they would need to inspect XmlNodes to know the type. Despite the name, it could be a collection of string representations of the xml nodes.